common/src/View/MapFrame.cpp

```diff
@@ -1286,7 +1286,15 @@ namespace TrenchBroom {
             if (canRenameSelectedGroups()) {
                 auto document = kdl::mem_lock(m_document);
                 assert(document->selectedNodes().hasOnlyGroups());
-                const std::string name = queryGroupName(this);
+
+                std::string suggestion;
+                if (document->selectedNodes().groupCount() == 1) {
+                    suggestion = document->selectedNodes().groups().front()->name();
+                } else {
+                    suggestion = "Multiple";
```

Comment:
I don't think I would put "Multiple" as the suggestion here. We usually use the `placeholderText` property of `QTextEdit` to show this info, so just putting it into the dialog like this deviates a bit from our usual approach. I think in this case I'd either put the name of the first group, or no suggestion at all.

Reply:
I'll use the name of the first group. TBH it's kind of strange to allow renaming multiple groups at once anyway.